code42_connector.py

```diff
@@ -154,6 +181,48 @@ def _handle_list_high_risk_employees(self, param, action_result):
         action_result.update_summary({"total_count": total_count})
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    @action_handler_for("get_highrisk_employee")
+    def _handle_get_high_risk_employee(self, param, action_result):
+        username = param["username"]
+        user_id = self._get_user_id(username)
+
+        try:
+            response = self._client.detectionlists.high_risk_employee.get(user_id)
+            action_result.add_data(response.data)
+            action_result.update_summary({"is_high_risk_employee": True})
+        except Py42NotFoundError:
+            action_result.update_summary({"is_high_risk_employee": False})
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    @action_handler_for("add_highrisk_tag")
+    def _handle_add_high_risk_tag(self, param, action_result):
+        username = param["username"]
+        risk_tag = param["risk_tag"]
+        user_id = self._get_user_id(username)
+        response = self._client.detectionlists.add_user_risk_tags(user_id, risk_tag)
+        all_tags = response.data.get("riskFactors", [])
+        response["riskFactors"] = _convert_to_obj_list(all_tags, "tag")
+        action_result.add_data(response.data)
+        message = f"All risk tags for user: {','.join(all_tags)}"
+        return action_result.set_status(phantom.APP_SUCCESS, message)
+
+    @action_handler_for("remove_highrisk_tag")
+    def _handle_remove_high_risk_tag(self, param, action_result):
+        username = param["username"]
+        risk_tag = param["risk_tag"]
+        user_id = self._get_user_id(username)
+        response = self._client.detectionlists.remove_user_risk_tags(user_id, risk_tag)
+        all_tags = response.data.get("riskFactors", [])
+        response["riskFactors"] = _convert_to_obj_list(all_tags, "tag")
+        action_result.add_data(response.data)
+        message = (
```

Comment:
The status message for `add` is just the risk tags from the response and a nice message. I tried both a summary and status message and ultimately felt the status message looked prettier in this case and the risk tags are available in the data anyway

The `remove` command additionally will output that the `user has not tags` if the tags are empty. The tags list should never be empty in the case of `add`